src/software/thunderscope/gl/gl_widget.py

```diff
@@ -355,3 +357,27 @@ def toggle_measure_mode(self) -> None:
             self.add_layer(self.measure_layer)
         else:
             self.remove_layer(self.measure_layer)
+
+    def calc_orthographic_distance(self) -> float:
+        field = DEFAULT_EMPTY_FIELD_WORLD.field
+        buffer_size = 0.5
+        fov = 60  # Same as default fov of GLViewWidget
```

Comment:
For the view to be (almost) orthograhic and everything look 2D, we would need the fov to be very small and as a result the distance very large. `1.0` degrees which we had before is a pretty good value. Probably a good idea to make FOV_DEGREES a constant so it is shared between this function and the `setCameraParams` call.\

Here's an example of Perspective vs Orthographic (might need to open in a new page if youre using darkmode) 
![image](https://github.com/UBC-Thunderbots/Software/assets/28585597/4c4c69e1-ac47-4cea-bfeb-0c172b19a1c9)

Reply:
Thanks for the comments Nima! 

From the top view, I don't think there is a noticeable difference, the field looks 2D regardless of whether the fov is 1 degree or 60 degrees. This is what it looks like with fov 60 degrees:

![image](https://github.com/UBC-Thunderbots/Software/assets/45322584/63523bdc-98e4-44b3-a7fb-c93efc12f927)

The only issue I can see with fov being 60 degrees is that when we zoom in a lot, the robots appear a little distorted, since their height becomes significant:

![image](https://github.com/UBC-Thunderbots/Software/assets/45322584/16ea8bdb-8e92-41ed-92d8-ba10e6c5184c)

If that's something we would like to avoid then I can make the fov smaller, like 1 degree as you suggested.

Another difference between 1 degree and 60 degrees fov appears when we try to orbit. If the fov is 1 degree, we will transition into an isometric 3D view, where parallel lines appear parallel:

![image](https://github.com/UBC-Thunderbots/Software/assets/45322584/43c7acd2-091d-4e1d-9061-6852217f5344)

On the other hand, a 60 degree fov seamlessly transitions into our regular 3D view, which is a perspective view:

![image](https://github.com/UBC-Thunderbots/Software/assets/45322584/7d5e879b-c84a-441c-84c1-0adfa04a3f0d)

Let me know your thoughts on whether we'd rather have normal-looking robots when zoomed in, or if we'd like to be able to orbit directly into 3D view.

Reply:
I personally prefer the orthographic view being mainly used for just top down and the robots not looking distorted. When I want to orbit in 3D I generally resort to switching to one of the other views.

Reply:
You could have two different views: Top Down Orthographic and Top Down Perspective

Reply:
That's also a fair option if you guys think top down perspective is also helpful

Reply:
I only use top down orthographic tbh and then the 3d view tbh. Not sure what the value of perspective is

Reply:
I agree, top down orthographic is more helpful. I've made the changes.